fluid/PaddleCV/image_classification/dist_train/dist_train.py

```diff
@@ -65,11 +65,17 @@ def parse_args():
     add_arg('split_var',          bool, True,               "Split params on pserver.")
     add_arg('async_mode',         bool, False,              "Async distributed training, only for pserver mode.")
     add_arg('reduce_strategy',    str,  "allreduce",        "Choose from reduce or allreduce.")
-    add_arg('skip_unbalanced_data', bool, False,            "Skip data not if data not balanced on nodes.")
+    add_arg('skip_unbalanced_data', bool,   False,          "Skip data not if data not balanced on nodes.")
```

Comment:
default = True is more friendly?

Reply:
Will delete this arg, `skip_unbalanced_data` is a deprecated API in pyreader.